docs/customize/administration/administration_interface.md

```diff
@@ -0,0 +1,346 @@
+
+    name = "oaipmh_edit"
+    url = "/oai-pmh/<pid_value>/edit"
+    resource_config = "oaipmh_server_resource"
+    pid_path = "id"
+    api_endpoint = "/oaipmh/sets"
+    title = "Edit OAI-PMH set"
+
+    list_view_name = "OAI-PMH"
+
+    form_fields = {
+        "name": {"order": 2, "text": "Set name"},
+        "spec": {"order": 3, "text": "Set spec"},
+        "search_pattern": {"order": 4, "text": "Search query"},
+        "created": {"order": 5},
+        "updated": {"order": 6},
+    }
+```
+
+For the full attributes list and description visit [reference docs](../../reference/administration_reference.md)
+
+A view defined as shown above has to be registered in the application's or module's entrypoints:
+
+```bash
+vim invenio-rdm-records/setup.cfg
+```
+
+```ini
+[options.entry_points]
+invenio_administration.views =
+    invenio_rdm_records_oai_edit = invenio_rdm_records.administration.views.oai:OaiPmhEditView
+```
+
+
+### Create resource view
+
+Example create view definition:
+
+```python
+from invenio_administration.views.base import (
+    AdminResourceCreateView,
+)
+
+class OaiPmhCreateView(AdminResourceCreateView):
+    """Configuration for OAI-PMH sets create view."""
+
+    name = "oaipmh_create"
+    url = "/oai-pmh/create"
+    resource_config = "oaipmh_server_resource"
+    pid_path = "id"
+    api_endpoint = "/oaipmh/sets"
+    title = "Create OAI-PMH set"
+
+    list_view_name = "OAI-PMH"
+
+    form_fields = {
+        "name": {"order": 1, "text": "Set name"},
+        "spec": {"order": 2, "text": "Set spec"},
+        "search_pattern": {"order": 3, "text": "Search query"},
+    }
+
+```
+
+For the full attributes list and description visit [reference docs](../../reference/administration_reference.md)
+
+A view defined as shown above has to be registered in the application's or module's entrypoints:
+
+```bash
+vim invenio-rdm-records/setup.cfg
+```
+
+```ini
+[options.entry_points]
+invenio_administration.views =
+    invenio_rdm_records_oai_create = invenio_rdm_records.administration.views.oai:OaiPmhCreateView
+```
+
+
+## Custom view
+
+Invenio administration provides flexibility of adding the custom view (not resource based) - enabling the developer to create an admin page from scratch
+
+```python
+from invenio_administration.views.base import AdminView
+
+class MyCustomView(AdminView):
+    """My custom view."""
+
+    extension_name = None
+    category = None
+    template = "invenio_administration/index.html"
+    url = None
+    menu_label = None
+    order = None
+    icon = None
+```
+
+For the full attributes list and description visit [reference docs](../../reference/administration_reference.md)
+
+A view defined as shown above has to be registered in the application's or module's entrypoints:
+
+```bash
+vim invenio-rdm-records/setup.cfg
+```
+
+```ini
+[options.entry_points]
+invenio_administration.views =
+    my_custom_admin_page = my_module.administration:MyCustomView
+```
```

Comment:
I would be a bit more explicit explaining how to register entry points with the following changes:

1: Before proposing the "MyCustomView" example, state that this file should be created in an Invenio module and propose an example path (eg: invenio-rdm-records/invenio_rdm_records/administration_views/custom_view.py).
2: Make clear that the entry point should be defined in the setup of the module in which it the view was created (currently it seems that regardless of the module it should be defined in the invenio-rdm-records setup).
3: Take advantage of the example path given in 1 to state clear how the path relates to the entry point definition. 

```
invenio_administration.views =
my_custom_view = invenio_rdm_records.administration_views.custom_view:MyCustomView
```